src/__test__/integration/avm.spec.ts

```diff
@@ -1,34 +1,33 @@
-import { FluencePeer, setLogLevel } from '../../index';
-import { Particle } from '../../internal/Particle';
+import { FluencePeer } from '../../index';
 import { handleTimeout } from '../../internal/utils';
 import { registerHandlersHelper } from '../util';
 
 let peer: FluencePeer;
 
 describe('Avm spec', () => {
-    afterEach(async () => {
-        if (peer) {
-            await peer.stop();
-        }
+    beforeAll(async () => {
```

Comment:
We want `beforeEach` and `afterEach` here so that tests don't interfere with each other

Reply:
I used beforeAll just so the tests run a bit faster. These test should not interfere with each other anyways. What do you think?

Reply:
Agreed. Will use beforeEach and afterEach just in case